node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -416,6 +414,9 @@ impl<C: Context> ActiveValidator<C> {
         state: &State<C>,
         instance_id: C::InstanceId,
     ) -> Option<SignedWireUnit<C>> {
+        if value.is_none() && !panorama.has_correct() {
+            return None; // Wait for the first proposal before creating a unit without a value.
+        }
```

Comment:
Is there a reason to do this that is inherent to this PR or is it just for code readability?